render.go

```diff
@@ -12,6 +11,9 @@ import (
 func (c *Camera) Draw(screen *ebiten.Image) {
 	screen.Clear()
 
+	// draw textured floor
+	screen.ReplacePixels(c.floorLvl.horBuffer.Pix)
```

Comment:
I can take care of this:

// Deprecated: as of v2.4. Use WritePixels instead.
https://github.com/hajimehoshi/ebiten/blob/d31b0189a20dbc3db28ba814774533a22fef1c17/image.go#L977